.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           command: cabal new-build
       - run:
           name: Run tests
-          command: cabal run graphql-parser-test -- release
+          command: cabal new-test --test-show-details=streaming
```

Comment:
Don't we also want to pass the `release` flag to the test suite? Which determines testing parameters. (IIRC, not passing it defaults to `dev`)